src/controllers/list.controller.ts

```diff
@@ -0,0 +1,57 @@
+import { Request, Response } from "express";
+import List from "../models/List";
+import sequelize from "../config/sequelize";
+import { NotFoundError } from "../errors/NotFoundError";
+import { logAction } from "../utils/logsAction";
+import Task from "../models/Task";
+
+export const ListController = {
+  async getAll(req: Request, res: Response) {
+    const { boardId } = req.params;
+    const lists = await List.findAll({
+      where: { boardId },
+      include: { model: Task },
+    });
+    res.json(lists);
+  },
+
+  async create(req: Request, res: Response) {
+    const { title } = req.body;
+    const { boardId } = req.params;
+    const result = await sequelize.transaction(async (transaction) => {
+      const list = await List.create({ title, boardId }, { transaction });
+      await logAction("List", list.id, "create", +boardId, list.title);
```

Comment:
Now that's where a transaction would work nicely, since log insert is an another write operation. But at the same time, you don't want to rollback the main action if a log insert fails, so it's usually done separately (sometimes even not `await`ed)